test/.gitignore

```diff
@@ -12,5 +12,8 @@ custom-unittest/custom-unittest
 path-subpackage-ref/test
 subpackage-ref/test
 subpackage-common-with-sourcefile-globbing/mypackage*
+version-spec/**/CMakeLists.txt
+version-spec/**/foo.cmake
+version-spec/**/foo
 
-/test_registry
+/test_registry
```

Comment:
Looks accidental too

Reply:
cmake files and `foo` binary generates by `version-spec.sh` test, why it don't be in .gitignore?

Reply:
The comment was about the removed newline.